applications/nrf5340_audio/broadcast_sink/main.c

```diff
@@ -451,9 +452,14 @@ int main(void)
 
 	LOG_DBG("nRF5340 APP core started");
 
+	channel_assignment_init();
+
 	ret = nrf5340_audio_dk_init();
 	ERR_CHK(ret);
 
+	ret = audio_system_init();
```

Comment:
These two calls are now done in all four mains. Discuss if we can put this somewhere more generic.

Reply:
It should be moved into nrf5340_audio_common_init then if is there for all four main files

Reply:
I moved `audio_system_init()` out of `nrf5340_audio_dk_init()` as it is needed regardless if the application is running on the audio DK or not, so I think it makes sense to have it in each `main.c`. Either way it won't matter functionally in this PR, so if you think it's better to keep `nrf5340_audio_dk` as-is I can move it back and we can discuss the topic when the application has full support for other DKs

Reply:
Ignore previous comment, I've moved `audio_system_init()` into `nrf5340_audio_common_init()`